src/api.ts

```diff
@@ -118,15 +118,26 @@ export class OpsgenieApi implements Opsgenie {
     if (!resp.ok) throw new Error(`Request failed with ${resp.status}: ${resp.statusText}`);
   }
 
-  async getAlerts(opts?: AlertsFetchOpts): Promise<Alert[]> {
+  async getAlerts(opts?: fetchOpts): Promise<Alert[]> {
     const limit = opts?.limit || 50;
+    const sort = opts?.sort || 'createdAt';
+    const order = opts?.order || 'desc';
     const query = opts?.query ? `&query=${opts?.query}` : '';
-    const response = await this.fetch<AlertsResponse>(`/v2/alerts?limit=${limit}${query}`);
 
-    return response.data;
+    let response = await this.fetch<AlertsResponse>(`/v2/alerts?limit=${limit}&sort=${sort}&order=${order}${query}`);
+    let alerts = response.data;
+
+    while (response.paging.next) {
```

Comment:
This will change the behavior of the `EntityOpsgenieAlertsCard` component.
It currently displays only a maximum of 3 alerts for a given entity, but this change will make it display every alerts related to that entity.

This is because AFAIK, the `limit` parameter accepted by Opsgenie applies to a result page, not the overall search.